streaq/types.py

```diff
@@ -1,13 +1,30 @@
+from __future__ import annotations
```

Comment:
This is unneeded as the library is explicitly Python 3.10+

Reply:
I did not add it for the union syntax, but for implicit usage of forward references.

You can read more about it in PEP649 and more importantly PEP749 which goes into detail about deprecating `from __future__ import annotations`.

In short, it allows me to write `def __call__(self, fn: CronTaskFn[WD, R]) -> RegisteredCron[WD, R]: ...`instead of `def __call__(self, fn: CronTaskFn[WD, R]) -> "RegisteredCron[WD, R]": ...`

It basically eliminates the need to escape forward references with quotes.

If you prefer using quotes instead, I can remove the import.

Reply:
I did not add it for the union syntax, but for implicit usage of forward references.

You can read more about it in PEP649 and more importantly PEP749 which goes into detail about deprecating from __future__ import annotations.

In short, it allows me to write def __call__(self, fn: CronTaskFn[WD, R]) -> RegisteredCron[WD, R]: ...instead of def __call__(self, fn: CronTaskFn[WD, R]) -> "RegisteredCron[WD, R]": ...

It basically eliminates the need to escape forward references with quotes.

If you prefer using quotes instead, I can remove the import.

Reply:
I do prefer quotes and use them elsewhere!